src/material/autocomplete/autocomplete-trigger.ts

```diff
@@ -632,6 +632,7 @@ export abstract class _MatAutocompleteTriggerBase implements ControlValueAccesso
 
     this.autocomplete._setVisibility();
     this.autocomplete._isOpen = this._overlayAttached = true;
+    this.autocomplete._formFieldLabelId = this._formField?._labelId;
```

Comment:
@crisbeto we're seeing this cause a ChangedAfterChecked error in Google because change detection runs for the first time during the `attach` call above, but then this changes `_formFieldLabelId` (and thus the `aria-labelledby expression) in the check pass. I can think of a couple hacky ways to work around this, but nothing super clean. Any ideas?